src/game/scenes/newsroom.c

```diff
@@ -392,7 +392,8 @@ int newsroom_create(scene *scene) {
             local->challenger = fight_stats->challenger;
             p2->pilot = local->challenger;
             fight_stats->challenger = NULL;
-        } else if(p1->chr) {
+        } else if(p1->chr && p2->pilot && p2->pilot->rank != 0) {
+            // we didn't just beat an unranked challenger
```

Comment:
that comment is a bit awkward. perhaps:
```suggestion
            // our match was against a ranked opponent
```